config/pmd-test.xml

```diff
@@ -153,8 +153,10 @@
             <property name="violationSuppressXPath"
                       value="//ClassOrInterfaceDeclaration[@Image='JavadocTokenTypesTest']
                                  //MethodDeclaration[@Name='testTokenValues']
-            | //ClassOrInterfaceDeclaration[@Image='GeneratedJavadocTokenTypesTest']
```

Comment:
The method `GeneratedJavaTokenTypesTest#testTokenNumbering` violates PMD rule "no more then 11 asserts per method", but was not suppressed. For some reason, PMD did not report violation till the test was converted to Junit5.

After migration to Junit5 violations appeared. And were suppressed.